wallet/src/main/scala/org/bitcoins/wallet/LockedWallet.scala

```diff
@@ -150,6 +151,18 @@ abstract class LockedWallet
     val updated = utxos.map(_.copyWithState(TxoState.Reserved))
     spendingInfoDAO.updateAll(updated)
   }
+
+  override def updateTxoStatesFromHeader(
```

Comment:
So this seems a little confusing to me, from looking at the method name, it seems that this would update all relevant utxos in the db, while looking at the implementation it only updates pending ones.

Reply:
The pending ones are the only relevant ones here, the only thing we can get from a header is a new chain tip, which means txs only care if they now have more confirmations.

Reply:
From looking at the method name/documentation this isn't apparent. Can you please rename it and add a scaladoc. I.e. `updateUtxoPendingStates()`